recipes/zope.hookable/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "zope.hookable" %}
+{% set version = "5.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f325838dbac827a1e2ed5d482c1f2656b6844dc96aa098f7727e76395fcd694
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+
+requirements:
+  build:
+    - python
```

Comment:
Please remove `python` and `pip` from the `build` section - they need to be in `host`, and only the compiler should stay here.